src/app/shared/store/user.actions.ts

```diff
@@ -222,6 +222,18 @@ export class OnUpdateWorkshopSuccess {
   static readonly type = '[user] update Workshop success';
   constructor(public payload) { }
 }
+export class UpdateStatus {
+  static readonly type = '[user] update Status';
+  constructor(public payload: WorkshopStatus) { }
+}
+export class OnUpdateStatusSuccess {
+  static readonly type = '[user] update Status success';
+  constructor(public payload) { }
```

Comment:
can you please also add type for the payload

Reply:
done